.env.sample

```diff
@@ -17,6 +17,7 @@ VITE_DISCORD_REDIRECT=http://localhost:3000
 VITE_RECAPTCHA_SITEKEY=6Lfqm6MeAAAAAFS5a0vwAfTGUwnlNoHziyIlOl1s
 VITE_TRADER_CONTRACT=0xbE005Fd6328960413f2627Ebb489B09507B2AEdF
 VITE_TREASURY_ADDRESS=0x1DB57407EE80709D4d862fe81399FBB35B8B9586
+VITE_FROG_CONTRACT=0x52A5ddFF59E0A27a1cBA75c8045A434b7D296A6c
```

Comment:
does not match with the one in testnet.yaml

Reply:
Updated this one, thanks!